File: app/features/document_intelligence/fixtures/simple.rb
# Quarterly infrastructure report helper.

class RegionReport
  attr_reader :name, :nodes, :utilization

  def initialize(name, nodes, utilization)
    @name = name
    @nodes = nodes
    @utilization = utilization
  end

  def healthy?
    utilization < 0.85
  end
end

File: app/features/document_intelligence/fixtures/simple.php
<?php
// Quarterly infrastructure report helper.

class RegionReport
{
    public function __construct(
        private string $name,
        private int $nodes,
        private float $utilization
    ) {
    }

    public function isHealthy(): bool
    {
        return $this->utilization < 0.85;
    }
}
